assignment2/src/com/assignment2/SaveUserDetails.java

```diff
@@ -0,0 +1,198 @@
+package com.assignment2;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Scanner;
+
+
+public class SaveUserDetails implements Serializable{
+	private static final long serialVersionUID = 1113799434508676095L;
+	public User userInput() {
+		Scanner s = new Scanner(System.in);
+		User user = new User();
+		while(true) {
+			System.out.println("Enter Full Name:");
+			String fullName = s.nextLine();
+			if(fullName.isEmpty())
```

Comment:
If and else blocks should always have braces and proper spacing.

Reply:
Okay